routes/error-tracker.js

```diff
@@ -180,6 +179,12 @@ function getHandler(req, res) {
   }
 
   let exception = params.s;
+  if (ignoreMessageOrException(params.m, exception)) {
+    res.set('Content-Type', 'text/plain; charset=utf-8');
+    res.status(statusCodes.BAD_REQUEST);
+    res.send('IGNORE\n').end();
+    return;
+  }
   // If format does not end with :\d+ truncate up to the last newline.
   if (!exception.match(/:\d+$/)) {
```

Comment:
I don't believe this block is necessary anymore, with the `standardizeStackTrace` call.